package ch.cern.todo.core.application.exception;

public class TaskRecordsMappedException extends RuntimeException {

    public TaskRecordsMappedException(String message) {
        super(message);
    }

}
